src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider specifying a fallback font-family in case the main one doesn't load. For example: `font-family: Roboto, Arial, sans-serif;`.